configure

```diff
@@ -99,6 +99,26 @@ check_do_prerequisites()
 	doctl compute domain records list "$DOMAIN" >/dev/null 2>&1 && msg check "  DO is managing $DOMAIN" || { errmsg "  error: domain not found. Please delegate '$DOMAIN' to DO's DNS service."; exit -1; }
 }
 
+check_aws_prerequisites()
+{
+	# see if we have eksctl
+	hash eksctl 2>/dev/null && msg check "  found eksctl" || { errmsg "  error: no eksctl on path. Install eksctl."; exit -1; }
+
+	hash aws 2>/dev/null && msg check "  found aws" || { errmsg "  error: no aws on path. Install AWS CLI tools."; exit -1; }
+	
+	# see if we are authenticated
+	aws sts get-caller-identity >/dev/null 2>&1 && msg check "  logged into AWS" || { errmsg "  error: issues authenticating to AWS..."; exit -1; }
+
+	# see if the domain has been properly set up
+	DOMAIN=$(echo "$HUB_FQDN" | cut -d . -f 2-)
+	output=$(aws --output=json route53 list-hosted-zones | jq --arg DOMAIN "${DOMAIN}." ' .HostedZones | .[] | select(.Name==$DOMAIN) ')
+	if [[ -n $output ]]; then
+	  msg check "  AWS is managing $DOMAIN"
```

Comment:
Indentation seems off (tab followed by two spaces -- should be two tabs).

Reply:
Fixed.